geth/node/node.go

```diff
@@ -1,261 +1,17 @@
 package node
 
 import (
-	"encoding/json"
-	"errors"
-	"fmt"
-	"os"
-	"path"
-	"path/filepath"
-	"strings"
+	"sync"
```

Comment:
I personally do not think there is much need for a file name change, `MakeNode` can't really be considered a `helper` here, we can in all sense categories it with names like `NewNode`, also other methods are most unexported which should not in anyway affect readability, am I missing something here?